### PR TITLE
fix: 🤔 sd-display main size variant

### PR DESCRIPTION
**Pull request by yoezlem**
## Description:
The default size variant in code is now changed to 4xl. 
Closes https://github.com/solid-design-system/solid/issues/1355.

## Definition of Reviewable:
- [x] relevant tickets are linked

### CODE REVIEW

**Comment by karlbaumhauer** [comment]
@yoezlem isn't that a breaking change?

**Comment by yoezlem** [comment]
> @yoezlem isn't that a breaking change?

not sure 🤔

@mariohamann @smfonseca ?

**Comment by auroraVasconcelos** [comment]
> > @yoezlem isn't that a breaking change?
> 
> not sure 🤔
> 
> @mariohamann @smfonseca ?

Hey @yoezlem I believe this is a breaking change considering that if you are using the component with the default size you would need to change the code since the default size is now different. 

(Issue link)[https://github.com/solid-design-system/solid/issues/1355] we opened an issue for this when we start re-writting this component, probably we have duplicated issues now fixed by this PR.

**Comment by smfonseca** [comment]
Yes, this is a breaking change imo. It will affect the visual design of the applications where the component is used.

**Comment by yoezlem** [comment]
Changed main branch to feat/breaking-changes

File: packages/components/src/styles/display/display.stories.ts [hunk 1]
```diff
@@ -40,17 +40,17 @@ export const Default = {
 /**
  * Use the `&--size-*` classes for alternative appearances.
 
- * - `sd-display--size-4xl`: 4xl can be used as an alternative
+ * - `sd-display`: 4xl is the default display size
```
Comment (auroraVasconcelos) [inline packages/components/src/styles/display/display.stories.ts:43]:
```suggestion
 * - 4xl is the default size
```

File: packages/components/src/styles/display/display.stories.ts [hunk 2]
```diff
@@ -40,17 +40,17 @@ export const Default = {
...
  * - `sd-display--size-3xl`: 3xl can be used as an alternative
```
Comment (auroraVasconcelos) [inline packages/components/src/styles/display/display.stories.ts:44]:
```suggestion
 * - `sd-display--size-3xl`
```

File: packages/components/src/styles/display/display.stories.ts [hunk 3]
```diff
@@ -40,17 +40,17 @@ export const Default = {
...
- * - xl is the default display size
+ * - `sd-display--size-xl`: xl can be used as an alternative
```
Comment (auroraVasconcelos) [inline packages/components/src/styles/display/display.stories.ts:45]:
```suggestion
 * - `sd-display--size-xl`
```

**Review by auroraVasconcelos — APPROVED** [review]
Just to keep consistency with the rest of docs

File: packages/components/src/styles/display/display.stories.ts (hunk 3 above)
Comment (yoezlem) [reply packages/components/src/styles/display/display.stories.ts:45]: Did we removed the explanation of different variants/classes etc.?

**Review by MartaPintoTeixeira — APPROVED** [review]

**Pull request merged**